docs/client-sdk/javascript/tutorials/filter-conversations.md

```diff
@@ -0,0 +1,129 @@
+---
+sidebar_label: Filter conversations
+sidebar_position: 3
+---
+
+# Filter conversations using conversation IDs and metadata
+
+With XMTP, a conversation is a set of messages sent between two blockchain account addresses (addresses). Two addresses can have multiple ongoing conversations.
+
+**Optionally**, you can assign an ID and metadata to a conversation to use as conversation filters in your app.
+
+For example, you can use conversation IDs and metadata to filter conversations by:
+
+* An app  
+Filter to display only conversations created by a specific app
+
+* A message type
+Filter to display only conversations that are notifications or alerts, for example. Conversation IDs are different from [content types](/docs/dev-concepts/content-types). Using a custom content type does not require that you use conversation IDs.
+
+* A subject line  
+Filter to display only conversations with a specific subject line, as in email
+
+* An NFT  
+Filter to display only conversations about an NFT, though not based on ownership of an NFT
+
+<!--What might this concrete NFT use case sound like? It is not based on ownership - but on a specific NFT contract address (NFT collection) or on a contract address and token ID - a specific instance of an NFT in a collection?-->
+
+Use this tutorial to learn how to use conversation IDs and metadata to filter conversations in your app.
+
+Conversation IDs and metadata are **not required**. Using conversation IDs affects the user experience in certain apps built with XMTP, so implement them only if you have a strategic need to filter conversations.
+
+<!--Only conversation Ids create separate conversation between two addresses, correct? Providing metadata only doesn't have this effect, correct?-->
```

Comment:
@neekolas any thoughts on the answer to this question? thank you...

Reply:
That's correct